packages/postgrest/lib/src/types.dart

```diff
@@ -88,11 +88,14 @@ enum CountOption {
   estimated,
 }
 
+// Not used anywhere. Will be removed in the next major version.
+// coverage:ignore-[start]
 /// Returns count as part of the response when specified.
 enum ReturningOption {
```

Comment:
What do you think about adding a `@Deprecated()` like I've done already with similar cases? Makes it easier in the future to find those again.